.prow.sh

```diff
@@ -1,5 +1,6 @@
 #! /bin/bash
 
+CSI_PROW_KUBERNETES_VERSION=latest
```

Comment:
Can you add a TODO, or open an issue to remove this once we add 1.16 jobs?

Reply:
opened

Reply:
See https://github.com/kubernetes-csi/external-resizer/issues/54